src/cli/commands.ts

```diff
@@ -128,6 +128,11 @@ export default {
           boolean: true,
           description: 'Skip packaging step',
           default: false
+        },
+        registries: {
+          array: true,
+          description:
+            'registries to publish, overriding your configuration in oc.json'
```

Comment:
`List of registries to publish to. This setting will take precedence over oc.json file.`

Reply:
done